src-gui/src/renderer/components/pages/swap/swap/init/walletAndMakers/WalletAndMakers.tsx

```diff
@@ -0,0 +1,184 @@
+import { Button, Typography, Box, Paper, Divider } from "@mui/material";
+import BitcoinQrCode from "renderer/components/pages/swap/swap/components/BitcoinQrCode";
+import ActionableMonospaceTextBox from "renderer/components/other/ActionableMonospaceTextBox";
+import MakerOfferItem from "./MakerOfferItem";
+import { useAppDispatch } from "store/hooks";
+import { usePendingSelectMakerApproval } from "store/hooks";
+import MakerDiscoveryStatus from "./MakerDiscoveryStatus";
+import { swapReset } from "store/features/swapSlice";
+import { TauriSwapProgressEventContent } from "models/tauriModelExt";
+import { SatsAmount } from "renderer/components/other/Units";
+import _ from "lodash";
+import CancelButton from "../../CancelButton";
+
+export default function WalletAndMakers({
+  deposit_address,
+  min_bitcoin_lock_tx_fee,
+  max_giveable,
+  known_quotes,
+}: TauriSwapProgressEventContent<"WaitingForBtcDeposit">) {
+  const dispatch = useAppDispatch();
+  const pendingSelectMakerApprovals = usePendingSelectMakerApproval();
+
+  const makerOffers = _.chain(pendingSelectMakerApprovals)
+    .map((approval) => ({
+      quoteWithAddress: approval.content.details.content.maker,
+      requestId: approval.content.request_id,
+    }))
+    .concat(
+      known_quotes.map((quote) => ({
+        quoteWithAddress: quote,
+        requestId: null,
+      })),
+    )
+    .sortBy((quote) => quote.quoteWithAddress.quote.price)
```

Comment:
Instead of only sorting by the price, sort using the logic in sortUtils.ts

We might have to refactor the function there to work with the types used here